Mendelugram/Mendelugram/View/Photo/PhotoTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PhotoTableViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class PhotoTableViewCell: UITableViewCell {
+
+    static let cellHeight: CGFloat = 430
+    static let reuseIdentifier = String(describing: PhotoTableViewCell.self)
+
+    struct Input {
+        let avatar: UIImage
+        let authorName: String
+        let locationName: String?
+        let photo: UIImage
+    }
+
+    @IBOutlet private weak var avatarImageView: UIImageView!
+    @IBOutlet private weak var authorNameLabel: UILabel!
+    @IBOutlet private weak var locationNameLabel: UILabel!
+    @IBOutlet private weak var photoImageView: UIImageView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        selectionStyle = .none
```

Comment:
Can you move this into a setup or configure method?

Reply:
I don't see a point in moving a one line that should happen in awakeFromNib a to special configure() method that will be called from awakeFromNib anyway.

Reply:
You have 3 lines there :-) And from my point of view, it is nice to keep only one approach throughout a whole codebase i.e. when you setup anything either from viewDidLoad or from awakeFromNib it is always in a method called setup.